ACLWeb.js

```diff
@@ -140,6 +150,45 @@ function scrapeIndex(doc, items) {
 	}
 }
 
+function scrape(doc) {
+		var pageurl = doc.location.href;
+		if ((pageurl.lastIndexOf(".bib", pageurl.length - 4) === pageurl.length - 4) ||
```

Comment:
this is rather odd.
```
var ext = url.substr(url.length-4);
if (ext == '.bib' || ext == '.pdf') {
```
(since we're assuming here `#` and `?` are not going to be part of the url (which is probably reasonable)